blueprints/app/files/.eslintrc.js

```diff
@@ -27,7 +27,15 @@ module.exports = {
       env: {
         browser: false,
         node: true
-      }
+      },
+      plugins: ['node'],
+      rules: Object.assign(
```

Comment:
Is there a reason you didn't use `"extends": ["plugin:node/recommended"],`?

Reply:
@Turbo87 asked the same question in https://github.com/ember-cli/ember-cli/pull/7451#discussion_r152213361 just above, answer is that sadly no. You cannot use extends inside of overrides yet.

This is being tracked in eslint/eslint#8813.

Reply:
oh shoot. sorry I was on mobile and missed that.

Reply:
I was gonna say add a comment to that issue to explain the code, but I read through the issue and doesn't even look like they want to do it. So a comment probably wouldn't make sense.

Reply:
```js
rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
  // add your custom rules and overrides for node files here
}),
```

can we reformat this to this?

Reply:
FWIW, looks like eslint _did_ add support for `extends` in overrides (landed in https://github.com/eslint/eslint/pull/11554 which is included in eslint@6.0.0+). We should work to remove this "bizarreness" now....